src/package-fetcher.js

```diff
@@ -24,6 +24,11 @@ async function fetchOne(ref: PackageReference, config: Config): Promise<FetchedM
   const dest = config.generateHardModulePath(ref);
 
   const remote = ref.remote;
+
+  if (ref.patterns && ref.patterns[0]) {
+    remote.pattern = ref.patterns[0];
```

Comment:
I don't know if this is an appropriate place to add this information but `remote` is one of the only things passed to the fetcher. I initially had it in `defaultManifest` but that didn't seem right either.